giantswarm/prometheus-rules
# Update AlertManagerNotificationsFailing routing on Atlas slack for opsgenie integration

towards https://github.com/giantswarm/giantswarm/issues/28986

### Checklist

- [x] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [x] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### How to test this PR

1. scale down prometheus-meta-operator
2. edit `alertmanager-config` secret by setting a wrong opsgenie key
3. you can check quickly the alert rule is working `alertmanager_notifications_failed_total{integration="opsgenie"}`. You should have result > 0
4. wait for `AlertmanagerNotifyNotificationsFailing` firing in [#alert-atlas-test](https://gigantic.slack.com/archives/C04AT72MKNK)
